### ISSUE TITLE
Allow for non-Original AMA case type

### ISSUE THREAD

**Issue by lowellrex** [department-of-veterans-affairs/caseflow#11678]
Currently, we hard-code all [AMA `Appeal`s' case type to "Original"](https://github.com/department-of-veterans-affairs/caseflow/blob/0c1c4959bb5c21fb3bea9336fd620ae18bc2e6c6/app/models/appeal.rb#L77). This ticket exists to allow for us to identify AMA `Appeal`s as having other case types, [similar to how we do](https://github.com/department-of-veterans-affairs/caseflow/blob/0c1c4959bb5c21fb3bea9336fd620ae18bc2e6c6/app/models/vacols/case.rb#L23) with `LegacyAppeal`s.

Specifically, we will have `Vacate` appeal types with Motions to Vacate work. 

## Acceptance criteria
* [ ] Non-original AMA `Appeal` case types (CAVC remand, etc.) are properly identified as such in Caseflow

**Comment by Sjones352** [comment]
**Conversation I had with Lowell about this ticket** 

Legacy Appeals already have a variety of case types because they can originate from a variety of sources. I don't think AMA appeals yet have that variety of case type since we expect them to mostly be new appeals (hence the "Original" case type designation)

However! LP announced a few weeks (months?) back that CAVC had remanded an AMA appeal to the Board. That appeal should have a case type that is different from "Original" because it is a CAVC remand (perhaps it should have the "CAVC remand" case type).
I don't know if we track that information (that this appeal was remanded from CAVC) in Caseflow yet, so ticket 11678 may be blocked until we capture that information in Caseflow (the lit support team is definitely tracking that information though)

https://dsva.slack.com/archives/C6E41RE92/p1563291264235800

**Comment by lowellrex** [comment]
Just discovered this now, but on the BEAAM-specific queue page [we applied the "BEAAM" case type label](https://github.com/department-of-veterans-affairs/caseflow/blob/ff330507a24a63676b99571af13fa9d92e3a139e/app/models/serializers/work_queue/beaam_serializer.rb#L29) for all appeals.

Should "BEAAM" be a possible case type for AMA appeals? Or should all BEAAM appeals now be labeled with the case type of "Original" (this is the current behaviour of the application)?

**Comment by lowellrex** [comment]
_[What is this graph?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```
      1 | 
      2 | |||
      3 | |||||
      5 | ||||
      8 | 
```

This ticket does not include any front-end work.

Estimating this at 3. Possible 2 because it could be as small as adding a little bit of new code that does not have to interact much with old code. Possible 5 if this involves entirely new user flows, working with design and many different teams, user training, etc.

Add release notes.

**Comment by leikkisa** [comment]
Closed, see #13211
